### PR TITLE
Explicit logError() call in corner case of Flink statement submission 400 corner case.

### PR DESCRIPTION
**Pull request by jlrobins**
## Summary of Changes

- Explicit logError() call in corner case of Flink statement submission 400 error handling corner case.

## Any additional details or context that should be provided?

- Followup to #1791, #1578.

## Pull request checklist

Please check if your PR fulfills the following (if applicable):

##### Tests

- [ ] Added new
- [ ] Updated existing
- [ ] Deleted existing

##### Other

- [ ] All new disposables (event listeners, views, channels, etc.) collected as  for eventual cleanup?

- [ ] Does anything in this PR need to be mentioned in the user-facing [CHANGELOG](https://github.com/confluentinc/vscode/blob/main/CHANGELOG.md) or [README](https://github.com/confluentinc/vscode/blob/main/public/README.md)?
- [x] Have you validated this change locally by [packaging](https://github.com/confluentinc/vscode/blob/main/README.md#packaging-steps) and installing the extension `.vsix` file?
  ```shell
  gulp clicktest
  ```

### CODE REVIEW

**Review by shouples — APPROVED** [review]

File: src/commands/flinkStatements.ts [hunk 1]
```diff
@@ -261,11 +261,22 @@ export async function submitFlinkStatementCommand(
       if (objFromResponse && typeof objFromResponse === "object" && "errors" in objFromResponse) {
         const responseErrors: { errors: [{ detail: string }] } = objFromResponse;
         logger.error(JSON.stringify(responseErrors, null, 2));
-
         errorMessages = responseErrors.errors.map((e: { detail: string }) => e.detail).join("\n");
       } else {
-        // will be the raw error string.
+        // will be the raw error string. Wacky we couldn't parse it. Flink backend change?
```
[the remark this answers is not in the document]
Comment (jlrobins) [reply src/commands/flinkStatements.ts:266]: We like a little color around here, Copilot.
